packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,160 @@
+# Number Field Updates (IxD)
+
+## Overview
+
+### Background
+
+- [Bring number-column formatting options to nimble-number-field](https://github.com/ni/nimble/issues/1855)
+- [Number IxD (Figma)](https://www.figma.com/design/r2yGNQNVFdE7cBO9CyHmQx/Nimble---IxD?node-id=1603-3464)
+
+The `nimble-number-field` leverages
+[fast-foundation](https://github.com/ni/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/number-field/number-field.ts)
+for its behavior, so we've inherited much of the behavior, including max/min
+validation and localization.
+
+For supported languages (English, French, German, Japanese, Chinese), the
+_decimal separator_ does not align with user expectations in German. For
+example, `1000000.01`
+[should](https://randombits.dev/articles/number-localization/locale-list) render
+as `1000000,01` in German but currently renders as `1000000.01` across all
+locales.
+
+Additionally, clients have requested the ability to define max/min values for
+the `nimble-number-field` to support form validation without automatically
+coercing user-entered values to fit within the specified range.
+
+In comparison, the HTML `<input type="number">` natively supports locale-aware
+_decimal separators_ and does not coerce values to the min/max limit.
+
+This IxD specifies requirements for a numeric control that will natively support
+localization, coercion, decimal precision, and other formatting options.
+
+## Usage
+
+Number fields are used in three distinct ways:
+
+1. **With Increment/Decrement Buttons (Steppers)**
+
+    - Suitable for small, controlled ranges of values where users adjust numbers
+      incrementally (e.g., specifying a slot number).
+    - Discrete integers with a predictable range.
+    - Step size is typically 1 but can vary (e.g., 5, 10).
+    - Max/min constraints are enforced, and values are coerced to stay within
+      range.
+    - _`nimble-number-field` supports this use case today._
+
+2. **Integer Input (Without Steppers)**
+
+    - Used when users need to input whole numbers without a constrained range.
+    - Validates input to ensure only integers are accepted (e.g., rejects 3.5 or
+      “abc”).
+    - May include min/max constraints (e.g., age between 0 and 150).
+
+3. **Decimal Values**
+    - Used for precise, fractional numbers (e.g., measurements or scientific
+      data).
+    - Defines decimal precision (e.g., 2 places for currency, 3 for scientific
+      data) and coerces values to enforce it.
+    - Does not use increment/decrement buttons.
+    - Supports locale-specific formatting (e.g., 1.23 vs. 1,23).
+
+## Requirements
+
+1. Support locale-specific formatting for decimal values (e.g., 10212,456 in
+   German).
+2. Allow specifying decimal precision (e.g., 2 places → 123.45).
+3. Provide validation-only mode for max/min, supporting both coercion and
```

Comment:
For the non-coercion mode, are there any features missing from the existing numeric? A client can set no `min` or `max`, subscribe to the value change event, and put it in an error state if the value is out of range.